Makefile.one

```diff
@@ -367,15 +394,37 @@ $(OUTPUT_DIR)/tracee-ebpf: $(OUTPUT_DIR)/tracee.bpf.core.o $(TRACEE_EBPF_SRC) ./
 .PHONY: clean-tracee-ebpf
 clean-tracee-ebpf:
 	$(CMD_RM) -rf $(OUTPUT_DIR)/tracee-ebpf
+	$(CMD_RM) -rf .*.md5
 
 .PHONY: test-tracee-ebpf
 test-tracee-ebpf: $(OUTPUT_DIR)/tracee.bpf.core.o
+	$(MAKE) $(OUTPUT_DIR)/btfhub
 	$(GO_ENV_EBPF) $(CMD_GO) test \
 		-tags $(GO_TAGS_EBPF) \
 		-v \
 		./tracee-ebpf/... \
 		./cmd/tracee-ebpf/...
 
+
+# btfhub (expensive: only run if core obj changed)
+
+SH_BTFHUB = ./3rdparty/btfhub.sh
+
+.PHONY: btfhub
+.ONESHELL:
```

Comment:
I did not understand what you mean here. What was innocently slipped ? The script ? The ".ONESHELL" special target ?

Reply:
the `.ONESHELL`. it changes the way the makefile is processed. I know it might be a good change, but still not the normal behavior. So I'm suggesting to either reconsider it (if it's just for one target), or to move it to the top where it's visible to readers

Reply:
Hum, I'm using it a lot, specially in the fixes I made for this Makefile in the PR of dropping the non core BPF build. There is a commit there fixing a bunch of stuff in this Makefile. I think we can drop ONE SHELL entirely there to make the Makefile not to rely in that feature (which would be ok I suppose)

Reply:
Okay, so.. I'm using the .ONESHELL feature *a lot* now. For:

1. .check_%
2. .checklib_%
3. .checkver_$(CMD_CLANG)
4. .checkver_$(CMD_GO)
5. btfhub

It allows us to have more complex shell scripting within the Makefile, which is great for features to check things (since we're checking a bunch of stuff in the makefile now). I'd rather keep it and move it to the top, like you said. Doing it in a "makefileadjusts" PR.

Reply:
https://github.com/aquasecurity/tracee/pull/1276